rust/arrow/src/arch/avx512.rs

```diff
@@ -41,6 +41,155 @@ pub(crate) unsafe fn avx512_bin_or(left: &[u8], right: &[u8], res: &mut [u8]) {
     std::ptr::copy_nonoverlapping(s, d, std::mem::size_of::<__m512i>());
 }
 
+#[target_feature(enable = "avx512f")]
+pub(crate) unsafe fn avx512_vec_sort_i64_single<'a>(input: &[i64]) -> [i64; 8] {
```

Comment:
May you please add some comments to the functions (and/or inline) so it's a bit easier for us to follow the code. Thanks

Reply:
Added for each method also comments for the operation blocks.